scripts/test.sh

```diff
@@ -60,7 +60,7 @@ ensure_tab_free
 #ensure_trailing_whitespace_free
 
 echo ":: Updating dependencies..."
-cargo update
+#cargo update
```

Comment:
Why was this commented out?

Reply:
Whoops, just got committed accidentally - needed it to run tests offline.